src/assets/scss/components/layout/_app-dashboard.scss

```diff
@@ -2,33 +2,34 @@ $app-dashboard-top-nav-color: #2c3840;
 $app-dashboard-top-nav-height: 55px;
 
 .app-dashboard {
-  height: 100vh;
   display: flex;
   flex-direction: column;
+  height: 100vh;
 }
 
 .app-dashboard-body {
-  flex: 1 1 auto;
   display: flex;
+  flex: 1 1 auto;
 }
 
 .app-dashboard-top-nav-bar  {
-  display: flex;
   align-items: center;
-  justify-content: center;
   background: $app-dashboard-top-nav-color;
+  display: flex;
```

Comment:
I don't see for which reason should `display` be after `background`. This is not a random list of names, but properties with more or less similar effects, or even "group effects". Most of them can be grouped to be more easily readable.

In Foundation we (try to) follow the following CSS order: https://gist.github.com/ncoden/d42f55df7c7970f548a02cd3468f9c86